common/djangoapps/util/milestones_helpers.py

```diff
@@ -0,0 +1,151 @@
+"""
+Helper methods for milestones api calls.
+"""
+
+from django.utils.translation import ugettext as _
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+from milestones.api import (
+    get_course_milestones,
+    add_milestone,
+    add_course_milestone,
+    remove_course_milestone,
+    get_course_milestones_fulfillment_paths,
+    add_user_milestone,
+    get_user_milestones,
+)
+from django.conf import settings
+
+
+def add_prerequisite_course(course_key, prerequisite_course_key):
+    """
+    It would create a milestone for given course and set it as
+    requirement milestone for given course and set it a fulfilment
+    milestone for pre-requisite course.
+    """
+    # create a milestone
+    milestone = add_milestone({
+        'name': _('Course {} requires {}'.format(unicode(course_key), unicode(prerequisite_course_key))),
+        'namespace': unicode(prerequisite_course_key),
+        'description': _('System defined milestone'),
+    })
+    # add requirement course milestone
+    add_course_milestone(course_key, 'requires', milestone)
```

Comment:
Constant values are currently available in the release_review branch via api.get_milestone_relationship_types()